### DISCUSSION TITLE
forms feature request: "collapse" render mode

### DISCUSSION THREAD

**Discussion by howardjohn** [community/community#5271] [Projects and Issues]
Currently, we can render text as code in forms with `render`. This is great! But one thing that is also useful is to collapse by default. For example:

<details>

A lot of text! It may even be code...

</details>

This can help when users are pasting in largeish output (say 200 lines or so) where a file may be less convenient.

**Comment by Kixunil** [comment]
Yeah, this would be great for logs. I would like it as well - came here to post this exact thing.

**Reply by MelnikovAG** [reply]
Hello

**Comment by Bibo-Joshi** [comment]
Possible duplicate of #5250

**Reply by MelnikovAG** [reply]
R
